### DISCUSSION TITLE
[Kotlin] Add defaultValueKotlinPrimitive to fix code generation

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [kobylynskyi/graphql-java-codegen#538] [Show and tell]
`defaultValueKotlinPrimitive` method has been removed, kotlin cannot generate code.
@kobylynskyi 😅 ,I'll add it back

![image](https://user-images.githubusercontent.com/29496873/108457890-62a83980-72ae-11eb-9ea8-410cb8e630ae.png)

**Comment by kobylynskyi** [comment]
@jxnu-liguobin, Sorry, I accidentally removed it (and all unit tests passed)

**Reply by jxnu-liguobin** [reply]
> Sorry, I accidentally removed it (and all unit tests passed)

The test does not generate a builder by default. 😅 Missing this case
